CODEOWNERS

```diff
@@ -105,7 +104,7 @@
 /boards/arm/nucleo*/                      @erwango @ABOSTM @FRASTM
 /boards/arm/nucleo_f401re/                @idlethread
 /boards/arm/nuvoton_pfm_m487/             @ssekar15
-/boards/arm/qemu_cortex_a53/              @carlocaione
+/boards/arm64/                            @carlocaione
```

Comment:
We should have this be broken up by board.  Since we only have a small handful its easier to do this now.

Reply:
OK